SpotifyAPI/Local/SpotifyLocalAPI.cs

```diff
@@ -139,11 +140,18 @@ public Boolean Connect()
             return _rh.Init();
         }
 
-        public StatusResponse GetStatus()
+        /// <summary>
+        /// Gets the Spotify status
+        /// </summary>
+        /// <returns></returns>
+        public async Task<StatusResponse> GetStatus()
```

Comment:
`ElapsedTick` is run in another thread by default, so there is no need making it async. I will check this tomorrow and see if we can get a simple fix